releasenotes/notes/remove-deprecated-pulse-commands-18381c59264090dd.yaml

```diff
@@ -0,0 +1,23 @@
+---
+upgrade:
+  - |
+    The `qiskit.pulse.commands` module containing `Commands` was deprecated in
+    Terra release 0.13.0 and have now been removed. You will have to upgrade
+    your Pulse code if you were still using commands. For example::
+
+        # old
+        Command(args)(channel)
+
+        # new
+        Instruction(args, channel)
+
+    ::
```

Comment:
This seems odd, but maybe I don't know enough about what it is doing.

Reply:
I wanted to make two separate literal blocks